hacks/setup-env.sh

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_URL="http://REACT_APP_API_URL.env.variable.not:set"
```

Comment:
Shouldn't the script include the `#!/bin/bash` directive?

Reply:
no because this file is meant to be sourced, no shell will execute the code

Reply:
That's true, thanks